.circleci/cimodel/data/binary_build_data.py

```diff
@@ -71,6 +81,15 @@ def get_processor_arch_name(cuda_version):
     ],
 )
 
+WINDOWS_LIBTORCH_CONFIG_VARIANTS = OrderedDict(
+    wheel=['release'],
```

Comment:
Why do we need to specify these variables here?

The combination of `libtorch` and `wheel`/`conda` seems confusing to me.

Reply:
This is because we build release configuration for wheels and conda, release/debug for libtorch.

Reply:
Is it better to fold them into one job for libtorch?

Reply:
Sorry, I misunderstand your comment. Will try to remove them.